backend/app/api/endpoints/admin.py

```
@@ -1707,3 +1715,643 @@ async def delete_public_retriever(
+                False,
+                f"Bot '{bot_namespace}/{bot_name}' is not a public resource. Please create it as a public bot first.",
+            )
+
+    return (True, None)
+
+
+@router.get("/public-teams", response_model=PublicTeamListResponse)
+async def list_public_teams(
+    page: int = Query(1, ge=1),
+    limit: int = Query(20, ge=1, le=100),
+    db: Session = Depends(get_db),
```

Ruff (0.14.13)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)